app/scripts/views/mixins/avatar-mixin.js

```diff
@@ -5,14 +5,23 @@
 // helper functions for views with a profile image. Meant to be mixed into views.
 
 define([
+  'underscore',
   'lib/profile-errors',
   'models/profile-image'
-], function (ProfileErrors, ProfileImage) {
+], function (_, ProfileErrors, ProfileImage) {
   'use strict';
 
   return {
     initialize: function (options) {
       this.notifications = options.notifications;
+      if (this.notifications) {
```

Comment:
That's a lot of `this.notifications`, a local variable reference will minify a bit better.